stable/weave-cloud/templates/manifest.yaml

```diff
@@ -0,0 +1,402 @@
+            },
+            "email-address": "support@weave.works"
+          }
+    apiVersion: v1
+    kind: ServiceAccount
+  - metadata:
+      labels:
+        name: weave-flux-agent
+        app: weave-flux
+        weave-cloud-component: flux
+        weave-flux-component: agent
+      name: weave-flux-agent
+      annotations:
+        cloud.weave.works/launcher-info: |-
+          {
+            "server-version": "master-e733d54",
+            "original-request": {
+              "url": "/k8s.yaml?t=REDACTED",
+              "date": "Tue Mar 21 2017 11:33:13 GMT+0000 (UTC)"
+            },
+            "email-address": "support@weave.works"
+          }
+    spec:
+      template:
+        metadata:
+          labels:
+            name: weave-flux-agent
+            app: weave-flux
+            weave-cloud-component: flux
+            weave-flux-component: agent
+        spec:
+          containers:
+            - name: agent
+              image: 'quay.io/weaveworks/fluxd:0.1.0'
+              imagePullPolicy: IfNotPresent
+              args:
+                - '--token={{.Values.ServiceToken}}'
+          serviceAccountName: weave-flux
+      replicas: 1
+    apiVersion: extensions/v1beta1
+    kind: Deployment
+  - metadata:
+      labels:
+        name: weave-scope
+      name: weave-scope
+      annotations:
+        cloud.weave.works/launcher-info: |-
+          {
+            "server-version": "master-e733d54",
+            "original-request": {
+              "url": "/k8s.yaml?t=REDACTED",
+              "date": "Tue Mar 21 2017 11:33:13 GMT+0000 (UTC)"
+            },
+            "email-address": "support@weave.works"
+          }
+    apiVersion: v1
+    kind: ServiceAccount
+  - metadata:
+      labels:
+        name: weave-scope-agent
+        app: weave-scope
+        weave-cloud-component: scope
+        weave-scope-component: agent
+      name: weave-scope-agent
+      annotations:
+        cloud.weave.works/launcher-info: |-
+          {
+            "server-version": "master-e733d54",
+            "original-request": {
+              "url": "/k8s.yaml?t=REDACTED",
+              "date": "Tue Mar 21 2017 11:33:13 GMT+0000 (UTC)"
+            },
+            "email-address": "support@weave.works"
+          }
+    spec:
+      template:
+        metadata:
+          labels:
+            name: weave-scope-agent
+            app: weave-scope
+            weave-cloud-component: scope
+            weave-scope-component: agent
+          annotations:
+            scheduler.alpha.kubernetes.io/tolerations: >-
+              [{"key":"dedicated","operator":"Equal","value":"master","effect":"NoSchedule"}]
+        spec:
+          containers:
+            - name: agent
+              image: 'weaveworks/scope:1.2.1'
+              imagePullPolicy: IfNotPresent
+              args:
+                - '--no-app'
+                - '--probe.docker.bridge=docker0'
+                - '--probe.docker=true'
```

Comment:
Currently only Docker is supported. I am aware that in some setups the bridge interface may be something like `cbr0`, but we've not come across such setups recently.